sorc/build_forecast.sh

```diff
@@ -6,6 +6,7 @@ cwd=`pwd`
 if [ $target = hera ]; then target=hera.intel ; fi
 if [ $target = orion ]; then target=orion.intel ; fi
 if [ $target = jet ]; then target=jet.intel ; fi
+if [ $target = cheyenne ]; then target=cheyenne.intel ; fi
 
```

Comment:
@evankalina, as we discussed, you might can use the following to replace the code section below:

```
if [ $target = wcoss_cray ]; then 
  app=HAFS
else
  app=HAFS-ALL
fi
cd hafs_forecast.fd/tests

./compile.sh "$target" "-DAPP=$app -DCCPP_SUITES=FV3_HAFS_v0_gfdlmp_tedmf_nonsst,FV3_HAFS_v0_gfdlmp_tedmf,FV3_HAFS_v0_hwrf_thompson,FV3_HAFS_v0_hwrf -D32BIT=ON" 32bit YES NO
```

In this case, one will not need to rebuild the forecast executable to run the HAFS CDEPS coupling configuration.

Reply:
Great suggestion. I added this code and tested it for a data model run on Orion, and it is working.